libsolidity/inlineasm/AsmData.h

```diff
@@ -64,6 +65,7 @@ struct VariableDeclaration { SourceLocation location; std::string name; std::sha
 struct Block { SourceLocation location; std::vector<Statement> statements; };
 /// Function definition ("function f(a, b) -> (d, e) { ... }")
 struct FunctionDefinition { SourceLocation location; std::string name; std::vector<std::string> arguments; std::vector<std::string> returns; Block body; };
+struct ForLoop { SourceLocation location; Block preCondition; Statement expression; Block postCondition; Block body; }
```

Comment:
Only the second item is an actual condition. What about `Block pre; Statement condition; Block post; Block body; }`?